server/public/layout.css

```diff
@@ -15,6 +15,25 @@
   width: 50%;
 }
 
+.row-meal-exercise {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.row-meal-exercise a {
+  font-size: 35px;
+  margin: 10px;
+}
+
+.fa-edit {
+  color: blue;
+}
+
+.fa-trash-alt {
+  color: red;
+}
+
```

Comment:
It is bad practice to directly alter a class from a CSS library, it can cause confusion for other people working on a project or even yourself later on. A better approach would be to create classes to just change the text color, something like `text-blue` and `text-red` then use those classes in addition to the icon classes.

Reply:
I made the changes you requested![users-can-edit-jmeal-and-exercise-plans-2](https://user-images.githubusercontent.com/71291742/142942570-f01c5cf9-3d9a-451f-9864-751f29bef9e6.gif)